include/zenoh-pico/api/primitives.h

```diff
@@ -317,6 +317,14 @@ const char *zp_config_get(z_config_t *config, unsigned int key);
  */
 int8_t zp_config_insert(z_config_t *config, unsigned int key, z_string_t value);
 
+/**
+ * Constructs a known :c:type:`z_encoding_t`.
+ *
+ * Returns:
+ *   Returns the constructed :c:type:`z_encoding_t`.
+ */
+z_encoding_t z_encoding(z_encoding_prefix_t prefix);
+
 /**
```

Comment:
If we are providing the helpers for the encodings, then I would provide one to set the suffix as well.
To be considered if we also need the getters for the `prefix` and `suffix` in  `z_encoding_t`.

And in this case, both `prefix` and `suffix` in  `z_encoding_t` can be privatized.

Reply:
I've extended `z_encoding` to accept both `prefix` and `suffix` in the constructor. I believe there is no need to add getters for `z_encoding_t`.